src/main/java/com/uid2/shared/secure/GcpOidcAttestationProvider.java

```diff
@@ -37,8 +37,9 @@ public void attest(byte[] attestationRequest, byte[] publicKey, Handler<AsyncRes
             LOGGER.debug("Validating signature...");
             var tokenPayload = tokenSignatureValidator.validate(tokenString);
 
+            //TODO: set the enclaveId once the registration of oidc enclaves is done
             if(this.Validate(tokenPayload)){
-                handler.handle(Future.succeededFuture(new AttestationResult(publicKey)));
+                handler.handle(Future.succeededFuture(new AttestationResult(publicKey, "TODO")));
```

Comment:
Is this supposed to stay todo for now?

Reply:
Yes, as the GCP OIDC work is still in progress. I don't know what that will be yet